pkg/util/exec/exec.go

```diff
@@ -151,3 +144,13 @@ func terminate(cmd *exec.Cmd, errCh <-chan error) {
 		log.Warnf("external process didn't exit gracefully within %s: %q", killTimeout, cmd)
 	}
```

Comment:
remove this switch and simply have 
```
<-time.After(killTimeout)
if isProcessRunning(cmd.Process.Pid) {
	if e := cmd.Process.Kill(); e != nil {
		log.Warnf("failed to send SIGKILL to the external process %q: %v", cmd, e)
	}
}
_ = cmd.Process.Release()
```

then we could remove the defer above.

Reply:
per https://github.com/tetratelabs/getenvoy/pull/163/files#r607519680

Reply:
if we do that we need to override killTimeout otherwise the tests will hang. NP will do

Reply:
done